buildSrc/src/main/kotlin/RSocketTestServer.kt

```diff
@@ -0,0 +1,61 @@
+import org.gradle.api.*
```

Comment:
why is this in buildSrc

Reply:
this task should be configured in 2 places:
1. in `rsocket-test-server` to provide executable
2. in modules, which need to run this task - for now it's only used in `rsocket-transport-ktor` for JS. But also will be useful if/when we will implement plain ios(darwin) transport #111 

now adding dependencies on this `TestServer` is simple as adding call in a module. Before we need to be sure that we call `evaluationDependsOn` in depending module and adding new `module` name in `rsocket-test-server`